kod/object/active/holder/nomoveon/battler/player.kod

```diff
@@ -11870,6 +11880,21 @@ messages:
       return piAction;
    }
 
+   GetMaskOrHelmRsc()
```

Comment:
Let's make this a little more generic, so if something like scarves are added in the future, someone won't have to go around renaming tons of stuff.

Also, add a comment as to the spirit of this function.  I'm not quite sure why the head overlay is special; the comment will enlighten me.